# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: security/sandbox/chromium/sandbox/win/src/policy_engine_unittest.cc
# Commit: 370c40e49230
# Full Hash: 370c40e4923073a87845b57c1f4851bacd9663da
# Author: Bob Owen <bobowencode@gmail.com>
# Date: 2019-06-12 21:42:35
# Regressor Bug: 1552160
# File Overlap Count: 2
# Description:
#   Bug 1552160 Part 1: Roll-up of chromium sandbox update and mozilla patches to get a running browser. r=jld,aklotz,tjr,bobowen
#   
#   This updates security/sandbox/chromium/ files to chromium commit 84108231f6e6e0772fb9a4643679ce76aa771e67.
#   
#   Existing and new patches applied from security/sandbox/chromium-shim/patches/with_update/ to give a compiling and mostly working browser.
# ==============================================================================

diff -r bd5ecd72f76c -r 370c40e49230 security/sandbox/chromium/sandbox/win/src/policy_engine_unittest.cc
--- a/security/sandbox/chromium/sandbox/win/src/policy_engine_unittest.cc	Wed Jun 12 21:55:59 2019 +1200
+++ b/security/sandbox/chromium/sandbox/win/src/policy_engine_unittest.cc	Wed Jun 12 11:10:48 2019 +0100
@@ -46,18 +46,16 @@
   OpcodeFactory opcode_maker(policy, policy_sz - 0x40);
 
   // Add rule set #1
-  opcode_maker.MakeOpWStringMatch(FileNameArg,
-                                  L"c:\\documents and settings\\",
+  opcode_maker.MakeOpWStringMatch(FileNameArg, L"c:\\documents and settings\\",
                                   0, CASE_INSENSITIVE, kPolNone);
   opcode_maker.MakeOpNumberMatch(CreationDispositionArg, OPEN_EXISTING,
                                  kPolNone);
-  opcode_maker.MakeOpVoidPtrMatch(SecurityAttributes, (void*)NULL,
-                                 kPolNone);
+  opcode_maker.MakeOpVoidPtrMatch(SecurityAttributes, nullptr, kPolNone);
   opcode_maker.MakeOpAction(ASK_BROKER, kPolNone);
 
   // Add rule set #2
-  opcode_maker.MakeOpWStringMatch(FileNameArg, L".TXT",
-                                  kSeekToEnd, CASE_INSENSITIVE, kPolNone);
+  opcode_maker.MakeOpWStringMatch(FileNameArg, L".TXT", kSeekToEnd,
+                                  CASE_INSENSITIVE, kPolNone);
   opcode_maker.MakeOpNumberMatch(CreationDispositionArg, CREATE_NEW,
                                  kPolNegateEval);
   opcode_maker.MakeOpAction(FAKE_ACCESS_DENIED, kPolNone);
@@ -66,7 +64,7 @@
   const wchar_t* filename = L"c:\\Documents and Settings\\Microsoft\\BLAH.txt";
   uint32_t creation_mode = OPEN_EXISTING;
   uint32_t flags = FILE_ATTRIBUTE_NORMAL;
-  void* security_descriptor = NULL;
+  void* security_descriptor = nullptr;
 
   POLPARAMS_BEGIN(eval_params)
     POLPARAM(filename)
@@ -99,7 +97,7 @@
   EXPECT_EQ(POLICY_MATCH, pr);
   EXPECT_EQ(FAKE_ACCESS_DENIED, pol_ev.GetAction());
 
-  delete [] reinterpret_cast<char*>(policy);
+  delete[] reinterpret_cast<char*>(policy);
 }
 
 }  // namespace sandbox